Presentation.Web/app/utility/validation.ts

```diff
@@ -1,13 +1,22 @@
 ﻿module Kitos.Utility {
     export class Validation {
         static validateUrl(url: string): boolean {
-            if (url == null) {
+            if (url == null || url === undefined) {
```

Comment:
brug lodash i stedet for. din === undefined er ikke robust.
`_.isUndefined(url)`